platform/net/firewall_provider_linux.go

```diff
@@ -37,8 +37,10 @@ const (
 func SetupNatsFirewall(mbus string) error {
 	// return early if
 	// we get a https url for mbus. case for create-env
-	// we get an empty string. case for http_metadata_service (responsible to extract the agent-settings.json from the metadata endpoint)s
-	if mbus == "" || strings.HasPrefix(mbus, "https://") {
+	// we get an empty string. case for http_metadata_service (responsible to extract the agent-settings.json from the metadata endpoint)
+	// we find that v1cgroups are not mounted (warden stemcells)
+	availableV1Groups, _ := cgroups.V1()
```

Comment:
I'm a little uncomfortable tossing away the error; I've been bitten by that too many times. Maybe add an error check here?

Also, is there any testing we can easily do? It doesn't seem like we're using interfaces, so the answer is probably no, but it would be nice to have them.

Reply:
Yeah that makes  sense. Thanks for spotting this.